docs/portal/deployment_details.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: 'Deployment Details'
+sidebar_position: 4
+---
+# Deployment Details
+
+* Environment Variables (view secrets repository - portal.env.local file)
+    - GATEKEEPER_INTEGRATION_SECRET (needed to authenticate so that user can have special interactions with portal (sign up for event reminders) (TODO figure this out))
+    - FIRESTORE_CREDENTIALS (needed to access firestore database (I think event info/generation is stored here (TODO figure this out)))
```

Comment:
event info and generation are not stored here. We use firestore only to store/add/remove reminders for events that users have created

Reply:
event info and generation are stored inside the md files inside the db folder